libparsec/crates/platform_device_loader/src/lib.rs

```diff
@@ -396,6 +394,64 @@ fn load_available_device_from_blob(
     })
 }
 
+#[cfg_attr(target_arch = "wasm32", expect(dead_code))]
+fn encrypt_device(device: &LocalDevice, key: &SecretKey) -> Bytes {
+    let mut cleartext = zeroize::Zeroizing::new(device.dump());
+    let ciphertext = key.encrypt(&cleartext);
+    cleartext.zeroize();
```

Comment:
Isn't the point of `Zeroizing` to automatically `zeroize`?

Reply:
The line was removed